core/examples/luigi-sample-angular/e2e/tests/navigation.spec.js

```diff
@@ -33,6 +37,63 @@ describe('Navigation', () => {
     cy.get('.fd-app__sidebar').should('contain', 'Second Child');
   });
 
+  it('Check if active node is selected', () => {
```

Comment:
I would suggest to use `cy.visit` instead of navigating manually and refactor a little bit to make the test shorter and more readable. For example, we don't need to use `should('exist')` after `find` because `find` will fail itself if it does not find the element. Here is my suggestion:
```
  it('Check if active node is selected', () => {
    cy.visit('/projects');
    cy.get('.fd-shellbar')
      .contains('Projects')
      .should('have.class', 'is-selected');

    cy.visit('projects/pr1');
    cy.get('.fd-side-nav__subitem')
      .contains('Project Settings')
      .click()
      .should('have.class', 'is-selected');
  });

  it('Check if active node reloads page', () => {
    cy.visit('/projects/pr1/developers');
    cy.getIframeBody().then($iframeBody => {
      cy.wrap($iframeBody)
        .should('contain', 'Developers content')
        .find('[title="visitors: 1"]')
      cy.get('.fd-app__sidebar')
        .contains('Project Settings')
        .click();
      cy.get('.fd-app__sidebar')
        .contains('Developers')
        .click();
      cy.wrap($iframeBody)
        .find('[title="visitors: 2"]')
    });
    cy.get('.fd-app__sidebar')
      .contains('Developers')
      .click();
    cy.getIframeBody().then($iframeBody => {
      cy.wrap($iframeBody)
        .find('[title="visitors: 1"]')
    });
  });
```

Reply:
👍